### PR TITLE
Work around bug in opencv3 headers.

### PR DESCRIPTION
**Pull request by iphydf**
OpenCV 3.1 doesn't define cvRound in C, only in C++. Thus, we now need
to compile av_test as C++ code.

### CODE REVIEW

**Comment by Diadlo** [comment]
LGTM

**Comment by nurupo** [comment]
I assume we want to support OpenCV 3 then.

File: CMakeLists.txt [hunk 1]
```diff
@@ -288,6 +288,11 @@ if(NOT WIN32 AND BUILD_TOXAV AND SNDFILE_FOUND AND PORTAUDIO_FOUND AND OPENCV_FO
     ${OPENCV_LIBRARIES}
     ${PORTAUDIO_LIBRARIES}
     ${SNDFILE_LIBRARIES})
+  # Due to https://github.com/opencv/opencv/issues/6585, we need to compile
+  # av_test as C++ for newer OpenCV versions.
+  if(OPENCV_VERSION VERSION_GREATER 3)
```
Comment (nurupo) [inline CMakeLists.txt]: Shouldn't that be "greater or equal to" instead?

Comment (iphydf) [inline CMakeLists.txt]: I think 3.1 is greater than 3. I don't know whether 3 works, but we got the bug report for 3.1. I'll make it "not less than", which is greater equal.

Comment (iphydf) [inline CMakeLists.txt]: Done by the way.

Comment (nurupo) [inline CMakeLists.txt]: approved

**Pull request merged**